composable-tasks/src/config.rs

```diff
@@ -0,0 +1,34 @@
+multiversx_sc::imports!();
+
+use crate::external_sc_interactions;
+
+#[multiversx_sc::module]
+pub trait ConfigModule:
+    external_sc_interactions::pair_actions::PairActionsModule
+    + external_sc_interactions::router_actions::RouterActionsModule
+    + external_sc_interactions::wegld_swap::WegldWrapModule
+{
+    #[only_owner]
+    #[endpoint(setWrapEgldAddr)]
+    fn set_wrap_egld_address(&self, new_addr: ManagedAddress) {
+        self.wrap_egld_addr().set(new_addr);
+    }
+
+    #[only_owner]
+    #[endpoint(setrouterAddr)]
+    fn set_router_address(&self, new_addr: ManagedAddress) {
+        self.router_addr().set(new_addr);
+    }
+
+    #[only_owner]
+    #[endpoint(setPairAddrForTokens)]
+    fn set_pair_address_for_tokens(
+        &self,
+        first_token_id: &TokenIdentifier,
```

Comment:
Use actual variables here, not references. The storage pair_address_for_tokens is ok to have references for the keys, but for the endpoint use owned values.